File: src/programmers/level2/make_min_value_12941/Solution.java
package programmers.level2.make_min_value_12941;

import java.util.Arrays;

public class Solution {
    public int solution(int []A, int []B)
    {
        int answer = 0;

        Arrays.sort(A);
        Arrays.sort(B);

        for(int i=0; i<B.length; i++){
            answer += A[i] * B[B.length-1-i];
        }

        return answer;
    }

    public static void main(String[] args) {
        int[] A = {1, 4, 2}, B= {5, 4, 4};
        Solution s = new Solution();
        int result = s.solution(A, B);
        System.out.println(result);
    }
}

File: src/programmers/level2/make_min_value_12941/solution.py
def solution(A,B):
    answer = 0
    A.sort()
    B.sort(reverse=True)

    for item in zip(A,B):
        answer += item[0]*item[1]

    return answer
A = [1, 4, 2]
B = [5, 4, 4]

result = solution(A,B)

print(result)